### PR TITLE
update nextcloud-tor.md (fixes #507)

### LINKED ISSUE #507: Update latest treehouses image link in nextcloud-tor.md
### Problem
As of 4/14/20, the image download link in nextcloud-tor.md does not have the latest image version 124. Instead, it links to a page with the latest version being 122. 

 ### Steps to reproduce the problem
Visit "Step 7 - Nextcloud with Tor Tutorial" The link http://download.treehouses.io/ is under "Prepare Treehouse images".

 ### Screenshots

The link here
image

leads to this download page:
image

which is outdated as the download.md page shows this:
image

 ### Proposed solution
Change the treehouses image link in nextcloud-tor.md to: https://treehouses.io/#!pages/download.md

### PR DESCRIPTION
**Pull request by law-lin**
Fixes #507 

### Description
Updated download link in nextcloud-tor.md

### Raw.Githack preview link

http://raw.githack.com/law-lin-OLE/law-lin.github.io/nextcloud-tor-download-link/#!pages/vi/nextcloud-tor.md

### CODE REVIEW

**Review by LordJashin32 — APPROVED** [review]

File: pages/vi/nextcloud-tor.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 ## Prepare Treehouses image
 
-Download the latest treehouses image from http://download.treehouses.io, then use [balenaEtcher](https://etcher.io) to flash the image onto your SD card.
+Download the latest treehouses image from https://treehouses.io/#!pages/download.md, then use [balenaEtcher](https://etcher.io) to flash the image onto your SD card.
```
Comment (dogi) [inline pages/vi/nextcloud-tor.md]: @law-lin  please use a relative link instead of an absolute one?

Comment (law-lin) [reply pages/vi/nextcloud-tor.md]: @dogi Just pushed an update - please review. Here's the raw.githack link:

http://raw.githack.com/law-lin-OLE/law-lin-OLE.github.io/nextcloud-tor-download-link/#!pages/vi/nextcloud-tor.md

**Comment by LordJashin32** [comment]
Looks good

File: pages/vi/nextcloud-tor.md [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
 ## Prepare Treehouses image
 
-Download the latest treehouses image from http://download.treehouses.io, then use [balenaEtcher](https://etcher.io) to flash the image onto your SD card.
+Download the latest treehouses image [here](./../download.md), then use [balenaEtcher](https://etcher.io) to flash the image onto your SD card.
```
Comment (dogi) [inline pages/vi/nextcloud-tor.md]: `pages/download.md` is what we want

also @law-lin it would be awesome if you allowed us to edit your PR ;)

Comment (law-lin) [reply pages/vi/nextcloud-tor.md]: @dogi I see.
 How do I allow you to edit my PR? I looked it up and I don't see the "Allow edits from maintainers" option on this page.

Comment (LordJashin32) [reply pages/vi/nextcloud-tor.md]: Would be on your own repository page -> settings i think

**Review by dogi — APPROVED** [review]

**Pull request merged**